optuna/integration/_lightgbm_tuner/optimize.py

```diff
@@ -984,5 +989,54 @@ def _create_objective(
             self.lgbm_kwargs,
             self.best_score,
             step_name=step_name,
+            model_dir=self._model_dir,
             pbar=pbar,
         )
+
+    def get_best_booster(self) -> "lgb.CVBooster":
+        """Return the best cvbooster.
+
+        If the best booster cannot be found, :class:`ValueError` will be raised.
+        To prevent the errors, please save boosters by specifying
+        both of the ``model_dir`` and the ``return_cvbooster`` arguments of
+        :meth: `~optuna.integration.lightgbm.LightGBMTunerCV.__init__`,
```

Comment:
A nit, but the document is not rendered correctly.

![image](https://user-images.githubusercontent.com/3255979/94632275-18291f00-0305-11eb-8bdb-411a57e6575e.png)

```suggestion
        :meth:`~optuna.integration.lightgbm.LightGBMTunerCV.__init__`,
```

Reply:
Thank you for letting me know, I will quickly fix this. Sorry I just learnt this notation is for the document.